src/acf.py

```diff
@@ -185,6 +208,12 @@ def ros_setup(self):
         self.create_service(SetFloat, '/ACF/set_payload', self.set_payload)
         self.create_service(SetFloat, '/ACF/set_f_zero', self.set_f_zero)
         self.create_service(SetDuration, '/ACF/set_t_ramp', self.set_t_ramp)
+        if self.frequency > 0:
+            self.create_timer(1 / self.frequency, self.timer_callback)
+        if self.joint_name != "":
+            self.joint_state_pub = self.create_publisher(JointState, '/joint_states', 10)
```

Comment:
```suggestion
            self.joint_state_pub = self.create_publisher(JointState, 'joint_states', 10)
```

try to avoid creating publishers that publish to absolute topics. They're harder / impossible to remap.

Reply:
Fixed in [3584491](https://github.com/sam-xl/ferrobotics_acf/pull/2/commits/3584491a49bffc6eb0be84192b1b2dd8cc327bcb)